packages/flutter_tools/lib/src/usage.dart

```diff
@@ -71,6 +74,27 @@ class Usage {
     // out-of-process from flutter_tools?
     return _analytics.waitForLastPing(timeout: new Duration(milliseconds: 250));
   }
+
+  void printUsage() {
+    if (_printedUsage)
+      return;
+    _printedUsage = true;
+
+    final String versionString = FlutterVersion.getVersionString(whitelistBranchName: true);
+
+    printStatus('');
+    printStatus('''
+  ╔════════════════════════════════════════════════════════════════════════════════════════════════════╗
+  ║              Welcome to Flutter! - Flutter version $versionString - https://flutter.io             ║
```

Comment:
getVersionString can return variable width strings, we should probably figure out the spacing here on the fly rather than hard-coding it. Feel free to file a bug on me if you want me to take care of it.